environment-lock-linux.yml

```diff
@@ -0,0 +1,361 @@
+  - r-rocr=1.0_7=r35h6115d3f_1002
+  - r-rpart=4.1_15=r35hcdcec82_1
+  - r-rprojroot=1.3_2=r35h6115d3f_1002
+  - r-rstudioapi=0.11=r35h6115d3f_0
+  - r-rvest=0.3.5=r35h6115d3f_0
+  - r-sandwich=2.5_1=r35h6115d3f_1
+  - r-scales=1.1.0=r35h6115d3f_0
+  - r-selectr=0.4_2=r35h6115d3f_0
+  - r-shiny=1.4.0.2=r35h6115d3f_0
+  - r-shinyjs=1.1=r35h6115d3f_0
+  - r-sourcetools=0.1.7=r35he1b5a44_1001
+  - r-sparsem=1.78=r35h9bbef5b_0
+  - r-squarem=2020.2=r35h6115d3f_0
+  - r-stringi=1.4.3=r35h0357c0b_2
+  - r-stringr=1.4.0=r35h6115d3f_1
+  - r-survival=3.1_12=r35hcdcec82_0
+  - r-sys=3.3=r35hcdcec82_0
+  - r-testthat=2.3.2=r35h0357c0b_0
+  - r-th.data=1.0_10=r35h6115d3f_1
+  - r-tibble=3.0.1=r35hcdcec82_0
+  - r-tidyr=1.0.2=r35h0357c0b_0
+  - r-tidyselect=1.0.0=r35h6115d3f_0
+  - r-tidyverse=1.2.1=r35h6115d3f_1002
+  - r-timedate=3043.102=r35h6115d3f_1001
+  - r-tinytex=0.22=r35h6115d3f_0
+  - r-tweenr=1.0.1=r35h0357c0b_1001
+  - r-utf8=1.1.4=r35hcdcec82_1002
+  - r-uuid=0.1_4=r35hcdcec82_0
+  - r-vctrs=0.2.4=r35hcdcec82_0
+  - r-vipor=0.4.5=r35h6115d3f_1002
+  - r-viridis=0.5.1=r35h6115d3f_1003
+  - r-viridislite=0.3.0=r35h6115d3f_1002
+  - r-whisker=0.4=r35h6115d3f_0
+  - r-withr=2.2.0=r35h6115d3f_0
+  - r-xfun=0.13=r35h6115d3f_0
+  - r-xml=3.98_1.20=r35hcdcec82_1
+  - r-xml2=1.3.2=r35h0357c0b_0
+  - r-xtable=1.8_4=r35h6115d3f_2
+  - r-yaml=2.2.1=r35hcdcec82_0
+  - r-zeallot=0.1.0=r35h6115d3f_1001
+  - r-zoo=1.8_7=r35hcdcec82_0
+  - readline=8.1.2=h0f457ee_0
+  - requests=2.21.0=py37_1000
+  - scikit-learn=0.20.3=py37ha8026db_1
+  - scipy=1.2.1=py37h09a28d5_1
+  - seaborn=0.9.0=py_2
+  - send2trash=1.8.0=pyhd8ed1ab_0
+  - setuptools=59.8.0=py37h89c1867_1
+  - sip=4.19.8=py37hf484d3e_1000
+  - six=1.16.0=pyh6c4a22f_0
+  - sqlite=3.40.0=h4ff8645_0
+  - statsmodels=0.10.1=py37hc1659b7_2
+  - sysroot_linux-64=2.12=he073ed8_15
+  - terminado=0.17.1=pyh41d4057_0
+  - testpath=0.6.0=pyhd8ed1ab_0
+  - tk=8.6.12=h27826a3_0
+  - tktable=2.10=hb7b940f_3
+  - tornado=6.2=py37h540881e_0
+  - tqdm=4.31.1=py_0
+  - traitlets=4.3.3=pyhd8ed1ab_2
+  - typing-extensions=4.4.0=hd8ed1ab_0
+  - typing_extensions=4.4.0=pyha770c72_0
+  - urllib3=1.24.3=py_1
+  - wcwidth=0.2.6=pyhd8ed1ab_0
+  - webencodings=0.5.1=py_1
+  - wheel=0.38.4=pyhd8ed1ab_0
+  - widgetsnbextension=3.4.2=py37_1000
+  - xlrd=1.2.0=pyh9f0ad1d_1
+  - xorg-kbproto=1.0.7=h7f98852_1002
+  - xorg-libice=1.0.10=h7f98852_0
+  - xorg-libsm=1.2.3=hd9c2040_1000
+  - xorg-libx11=1.7.2=h7f98852_0
+  - xorg-libxau=1.0.9=h7f98852_0
+  - xorg-libxdmcp=1.1.3=h7f98852_0
+  - xorg-libxext=1.3.4=h7f98852_1
+  - xorg-libxrender=0.9.10=h7f98852_1003
+  - xorg-renderproto=0.11.1=h7f98852_1002
+  - xorg-xextproto=7.3.0=h7f98852_1002
+  - xorg-xproto=7.0.31=h7f98852_1007
+  - xz=5.2.6=h166bdaf_0
+  - zeromq=4.3.4=h9c3ff4c_1
+  - zipp=3.11.0=pyhd8ed1ab_0
+  - zlib=1.2.13=h166bdaf_4
+  - zstd=1.5.2=h3eb15da_6
+  - pip:
+    - hetio==0.2.11
+    - hetmatpy==0.0.0
+    - regex==2022.10.31
+    - xswap==0.0.2
```

Comment:
Noting that the environment.lock loses the git commit hash versioning from `environment.yml`